lib/adoptoposs/submissions.ex

```diff
@@ -151,6 +152,24 @@ defmodule Adoptoposs.Submissions do
     |> Repo.update()
   end
 
+  @doc """
+  Sets the status of a project.
+
+  ## Examples
+
+      iex> set_project_status(project, :published)
+      {:ok, %Project{}}
+
+      iex> update_project(project, :not_allowed_status)
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def set_project_status(%Project{} = project, status) do
```

Comment:
minor, but coming from a Ruby background, I always shyed away from using `set_` and `get_` prefixes, maybe something like `update_`?

Reply:
Yes, good point, update is probably better here. Phoenix generates `get_<sth>` functions e.g. when you create a context, but in terms of `set_…` I figure it’s more consistent if we stick to the `list, get, create, update, delete, change` prefixes. Changed it. :+1: